components/calendar/src/cal/hijri.rs

```diff
@@ -129,12 +129,18 @@ pub trait Rules: Clone + Debug + crate::cal::scaffold::UnstableSealed {
 
 /// [`Hijri`] [`Rules`] based on an astronomical simulation for a particular location.
 ///
-/// These simulations are unofficial and are known to not necessarily match sightings
-/// on the ground. Unless you know otherwise for sure, instead of this variant, use
-/// [`UmmAlQura`], which uses the results of KACST's Mecca-based calculations.
+/// The simulations use the relative positions of the Earth, moon, and sun to predict the
+/// exact moment a new moon occurs. Because this is rarely the instant when a crescent
+/// sighting occurs, the month start dates preducted by these rules will often be one or
+/// more days earlier than actually observed. Applications using these rules should have
```

Comment:
maybe say something like "These `Rules` shouldn't be used directly, but can form the basis for a `Rules` implementation based on human sightings". The correction should be applied at the `Rules` level, not at the `Date` level (but I don't really see the point of using this if you can just write a sighting `Rules`).